x-pack/plugins/reporting/common/constants.ts

```diff
@@ -45,6 +45,7 @@ export const KBN_SCREENSHOT_HEADER_BLOCK_LIST = [
 
 export const KBN_SCREENSHOT_HEADER_BLOCK_LIST_STARTS_WITH_PATTERN = ['proxy-'];
 
+export const UI_SETTINGS_SEARCH_INCLUDE_FROZEN = 'search:includeFrozen';
```

Comment:
cc @lukasolson shouldn't this setting be respected inside `search` service ? (and as such we should not need to set it here ?)

Reply:
Filed: https://github.com/elastic/kibana/issues/110679

Reply:
Looks like this setting is only respected with the `ESE_SEARCH_STRATEGY`. Unfortunately, CSV export can not use that search strategy because it doesn't support `scroll` queries: https://github.com/elastic/kibana/issues/110679#issuecomment-933736075